randovania/games/prime1/assets/hash_words.json

```diff
@@ -1,99 +1,138 @@
 [
-    "Samus",
-    "Metroid",
-    "Prime",
-    "Varia",
-    "Gravity",
-    "Phazon",
-    "Power",
-    "Wave",
-    "Ice",
-    "Plasma",
-    "Thermal",
-    "X-Ray",
-    "Tallon",
-    "Chozo",
-    "Magmoor",
-    "Phendrana",
-    "Mines",
-    "Crater",
-    "Frigate",
-    "Truth",
-    "Strength",
-    "Elder",
-    "Wild",
-    "Lifegiver",
-    "Warrior",
-    "Nature",
-    "Sun",
-    "World",
-    "Spirit",
-    "Newborn",
-    "Artifact",
-    "Glider",
-    "Zoomer",
-    "Shriekbat",
-    "Incinerator",
-    "Mecha",
-    "Flaahgra",
-    "Thardus",
-    "Omega",
-    "Pirate",
-    "Sheegoth",
-    "Wasp",
-    "Beetle",
-    "Blastcap",
-    "Bloodflower",
-    "Ghost",
-    "Drone",
-    "Crystallite",
-    "Elite",
-    "Eyon",
-    "Oculus",
-    "Fission",
-    "Hunter",
-    "Geemer",
-    "Grizby",
-    "Hive",
-    "Parasite",
-    "Burrower",
-    "Trooper",
-    "Bombu",
-    "Jelzap",
-    "Lumigek",
-    "Turret",
-    "Ridley",
-    "Exoskeleton",
-    "Essence",
-    "Puffer",
-    "Reaper",
-    "Scarab",
-    "Shadow",
-    "Triclops",
-    "Access",
-    "Transport",
-    "Tunnel",
-    "Radion",
-    "Cordite",
-    "Talloric",
-    "Bendezium",
-    "Station",
-    "Quarantine",
-    "Hydra",
-    "Aether",
-    "Shorelines",
-    "Chapel",
-    "Temple",
-    "Subchamber",
-    "Jump",
-    "Morph",
-    "Boost",
-    "Spider",
-    "Grapple",
-    "Super",
-    "Charge",
-    "Wavebuster",
-    "Spreader",
-    "Core",
-    "Flamethrower"
-]
\ No newline at end of file
+ 	"Access",
+	"Aether",
```

Comment:
IDK if it's just me, but I associate Aether waaay more with echoes than prime 1. Where is that even used there?

Reply:
IIRC, it was for Research Lab Aether. I'm happy to remove or replace it, though - it's fair that it's way more associated with Echoes.